tensorflow_addons/custom_ops/image/cc/kernels/euclidean_distance_transform_op.h

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_ADDONS_EUCLIDEAN_DISTANCE_OP_H_
+#define TENSORFLOW_ADDONS_EUCLIDEAN_DISTANCE_OP_H_
+
+#define EIGEN_USE_THREADS
+
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+
+#include "tensorflow/core/framework/tensor_types.h"
+#include "tensorflow/core/platform/types.h"
+
+#include <limits>
+
+#define IDX(x, y)                                                              \
+  Eigen::array<Eigen::DenseIndex, 4>({coords[0], x, y, coords[3]})
```

Comment:
IMHO this macro makes the code slightly less readable as it somehow "hides" the fact that coords is used.

Reply:
I'm using the macro only once in my code, so I'm just gonna remove it all together.